pkg/tcpip/network/fragmentation/reassembler.go

```diff
@@ -116,3 +131,23 @@ func (r *reassembler) checkDoneOrMark() bool {
 	r.mu.Unlock()
 	return prev
 }
+
+// TimeOut function generates ICMP TTL Error message (Fragment reassembly time exceeded message).
+func (r *reassembler) TimeOut(ra *stack.Route, netHeader buffer.View, vv buffer.VectorisedView) {
```

Comment:
Please rebase on HEAD again as a significant change has landed which wraps netheader/data etc in a single struct called tcpip.PacketBuffer.

Reply:
we have taken care of this comment.